src/commonMain/kotlin/com/shopify/library/internal/star/StarManager.kt

```diff
@@ -18,7 +17,16 @@ class StarManager(
     private val _printResult = MutableSharedFlow<Boolean>()
     val printResult = _printResult.asSharedFlow()
 
-    val printerStatus = starIoExtManagerWrapper.printerStatus
+    private val _printerStatus = MutableSharedFlow<String>()
+    val printerStatus = _printerStatus.asSharedFlow()
+
+    init {
+        backgroundScope.launch {
+            starIoExtManagerWrapper.printerStatus.collect { status ->
+                _printerStatus.emit(status)
+            }
+        }
+    }
```

Comment:
@Baileypollard I modified some code here in order to emit printer statuses from either the `StarIoExtManager` or by calling the `starSdk.getWifiPrinterStatus()` method. Can you take a quick look and verify this still works as expected in your case?